_data/toc.yaml

```diff
@@ -1272,6 +1272,26 @@ manuals:
           title: Run Docker Desktop for Windows in a VM or VDI environment
         - path: /desktop/uninstall/
           title: Uninstall Docker Desktop
+    - sectiontitle: Hardened Desktop
+      section:
+        - path: /desktop/hardened-desktop/
+          title: Overview
+        - sectiontitle: Admin controls (Early Access)
+          section:
+          - path: /desktop/hardened-desktop/admin-controls/
+            title: What is Admin Controls?
+          - path: /desktop/hardened-desktop/admin-controls/configure-ac/
+            title: Configure Admin controls
+        - sectiontitle: Enhanced Container Isolation (Early Access)
```

Comment:
we also need to remove 'early access' here (and anywhere else throughout the docs)